### ISSUE TITLE
SqliteParameter.SqliteType throws InvalidOperationException for DateOnly/TimeOnly

### ISSUE THREAD

**Issue by inosik** [dotnet/efcore#27484]
The `SqliteParameter.SqliteType` Property throws an `InvalidOperationException` when the value is a `DateOnly` or `TimeOnly`.

``` csharp
var p = new SqliteParameter { ParameterName = "date", Value = new DateOnly() };
_ = p.SqliteType;
```

``` text
System.InvalidOperationException: No mapping exists from object type System.DateOnly to a known managed provider native type.
   at Microsoft.Data.Sqlite.SqliteValueBinder.GetSqliteType(Object value)
   at Microsoft.Data.Sqlite.SqliteParameter.get_SqliteType()
```

`DateOnly` and `TimeOnly` are missing here:

https://github.com/dotnet/efcore/blob/f741ba44765d72aae410be743b0907ecec6d5a7a/src/Microsoft.Data.Sqlite.Core/SqliteValueBinder.cs#L240

### Version information

Microsoft.Data.Sqlite version: 6.0.2, 7.0.0-preview.1.22076.6
Target framework: .NET 6.0
Operating system: n/a

**Comment by roji** [comment]
Duplicate of #27213

**Comment by roji** [comment]
This is already fixed in PR #27291, which closes #27213.
